pipelines/data_preparation_pipeline/config_data_preparation_pipeline.yaml

```diff
@@ -5,3 +5,8 @@ steps:
     enable_cache: False
   validate_data:
     enable_cache: False
+  version_data:
+    parameters:
+      data_version_name: "first_version"
```

Comment:
If we run this pipeline again, would this required us to manually change the `data_version_name` in the config?